boards/DCM/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -0,0 +1,110 @@
+#include "App_SharedSetPeriodicCanSignals.h"
+#include "App_SetPeriodicCanSignals.h"
+#include "App_InRangeCheck.h"
+#include "configs/App_TorqueRequestThresholds.h"
+
+STATIC_DEFINE_APP_SET_PERIODIC_CAN_SIGNALS_IN_RANGE_CHECK(DcmCanTxInterface)
+
+// TODO: Implement PID controller to maintain DC bus power at 80kW
+// #680
+void App_SetPeriodicCanSignals_TorqueRequests(const struct DcmWorld *world)
+{
+    struct DcmCanRxInterface *can_rx = App_DcmWorld_GetCanRx(world);
+    struct DcmCanTxInterface *can_tx = App_DcmWorld_GetCanTx(world);
+
+    // Regen allowed when braking or (speed > 5kmh and AIRs closed)
+    const float regen_threshold_kph = 5.0f;
+    const bool  is_every_air_closed =
+        (App_CanRx_BMS_AIR_STATES_GetSignal_AIR_POSITIVE(can_rx) ==
+         CANMSGS_BMS_AIR_STATES_AIR_POSITIVE_CLOSED_CHOICE) &&
+        (App_CanRx_BMS_AIR_STATES_GetSignal_AIR_NEGATIVE(can_rx) ==
+         CANMSGS_BMS_AIR_STATES_AIR_NEGATIVE_CLOSED_CHOICE);
+    const bool is_vehicle_over_regen_threshold =
+        (App_CanRx_FSM_WHEEL_SPEED_SENSOR_GetSignal_LEFT_WHEEL_SPEED(can_rx) >
+         regen_threshold_kph) &&
+        (App_CanRx_FSM_WHEEL_SPEED_SENSOR_GetSignal_RIGHT_WHEEL_SPEED(can_rx) >
+         regen_threshold_kph);
+    const bool is_regen_allowed =
+        is_vehicle_over_regen_threshold && is_every_air_closed;
+
+    const float regen_paddle_percentage =
+        (float)App_CanRx_DIM_REGEN_PADDLE_GetSignal_MAPPED_PADDLE_POSITION(
+            can_rx);
+    float torque_request;
+
+    /* Calculating the torque request
+     *
+     * 1) If regen. braking is on, use the regen. paddle percentage
+     * 2) If regen. braking is off, use the accelerator pedal percentage
+     *
+     * - MAX_SAFE_TORQUE_REQUEST_NM = 21 Nm, the max torque the motor can
+     * provide
+     *
+     * 1) If regen. braking is allowed and the regen paddle is actuated,
+     *    the torque request (in Nm) is negative and given by:
+     *                                  (Regen Paddle Percentage)
+     *   Torque Request    =  (-1) * -------------------------------  *
```

Comment:
nit; either keep the constants as all caps or title case

Reply:
Brackets around Regen Paddle Percentage is also not necessary. I'm being nit here but I think it looks much nicer

```
(-) Regen Paddle Percentage
------------------------------
     Max Torque Request
```

Reply:
> nit; either keep the constants as all caps or title case

I did -- `MAX_SAFE_TORQUE_REQUEST` is the only constant in the expression